libtelemetry/src/test/java/com/mapbox/services/android/telemetry/AppUserTurnstileTest.java

```diff
@@ -23,6 +23,8 @@ public void checksAppUserTurnstileType() throws Exception {
 
   private Event obtainAnAppUserTurnstileEvent() {
     boolean indifferentTelemetryEnabled = false;
-    return new AppUserTurnstile(indifferentTelemetryEnabled, "anySdkIdentifier", "anySdkVersion");
+    String userId = TelemetryUtils.obtainUniversalUniqueIdentifier();
```

Comment:
This is not necessary. We could just pass in `"anyUserId"` 👇 and avoid universal unique identifier creation/generation.

The same thing happens with below changed tests.